src/main/java/seedu/address/logic/commands/CommandResult.java

```diff
@@ -17,21 +18,35 @@
     /** The application should exit. */
     private final boolean exit;
 
+    /** The application should display the window allowing the user to import a CSV file. */
+    private final boolean importFromCsv;
+
+    /** The application should display the window allowing the user to export to a CSV file. */
+    private final boolean exportToCsv;
+
+
     /**
      * Constructs a {@code CommandResult} with the specified fields.
+     * The HashMap can have keys for "showHelp", "exit", "importFromCsv", "exportToCsv".
+     * The default values are false.
      */
-    public CommandResult(String feedbackToUser, boolean showHelp, boolean exit) {
+    public CommandResult(String feedbackToUser, HashMap<String, Boolean> settings) {
```

Comment:
Nice how you incorporated a HashMap here compared to the previous constructor since the number of settings you've implemented increased! Makes everything look cleaner.